ts/Grid/Pro/CellEditing/CellEditingComposition.ts

```diff
@@ -141,14 +141,10 @@ namespace CellEditingComposition {
         this: TableCell,
         e: GridEvent<TableCell, KeyboardEvent>
     ): void {
-        if (e.originalEvent.key === 'Enter') {
+        if (e.originalEvent?.key === 'Enter') {
```

Comment:
I would replace it with `if (e.originalEvent?.key !== 'Enter') { return false }`